recipes/avahi/all/conanfile.py

```diff
@@ -1,8 +1,12 @@
-from conans import ConanFile, tools, AutoToolsBuildEnvironment
-from conans.errors import ConanInvalidConfiguration
 import os
 
-required_conan_version = ">=1.33.0"
+from conan import ConanFile
+from conan.tools.files import copy, get, rmdir, rm
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.52.0"
```

Comment:
```suggestion
required_conan_version = ">=1.47.0"
```
Do not need to enforce to the latest version